backend/src/main/java/kr/touroot/travelogue/service/TravelogueFacadeService.java

```diff
@@ -42,12 +43,12 @@ public class TravelogueFacadeService {
     private final MemberService memberService;
 
     @Transactional
-    public TravelogueResponse createTravelogue(MemberAuth member, TravelogueRequest request) {
+    public TravelogueCreateResponse createTravelogue(MemberAuth member, TravelogueRequest request) {
         Member author = memberService.getById(member.memberId());
         Travelogue travelogue = travelogueService.createTravelogue(author, request);
-        List<TagResponse> tags = travelogueTagService.createTravelogueTags(travelogue, request.tags());
-        TravelogueLikeResponse like = travelogueLikeService.findLikeByTravelogueAndLiker(travelogue, author);
-        return TravelogueResponse.of(travelogue, createDays(request.days(), travelogue), tags, like);
+        travelogueTagService.createTravelogueTags(travelogue, request.tags());
+        travelogueLikeService.findLikeByTravelogueAndLiker(travelogue, author);
+        return new TravelogueCreateResponse(travelogue.getId());
```

Comment:
의견)
```suggestion
        return new TravelogueCreateResponse(travelogue);
```
이렇게 넘겨서 `TravelogueCreateResponse`가 스스로 id를 꺼내쓰게 만들면 어떨까요?
FacadeService가 TravelogueCreateRsponse의 내부 구현을 아는 느낌 ....

Reply:
동의합니다! 고치겠습니다!